tox.ini

```diff
@@ -0,0 +1,13 @@
+# This file is part of DataCite.
+#
+# Copyright (C) 2015 CERN.
+#
+# DataCite is free software; you can redistribute it and/or modify it
+# under the terms of the Revised BSD License; see LICENSE file for
+# more details.
+
+[tox]
+envlist = py26, py27, py33
+
+[testenv]
+commands = {envpython} setup.py test
```

Comment:
On my machine, I get `tox` troubles:

```
$ tox -e py27
[...]
running build_ext
usage: setup.py [options] [file_or_dir] [file_or_dir] [...]
setup.py: error: unrecognized arguments: --clearcache --clearcache tests datacite
```

The fix is to list `pytest` dependencies explicitly in `tox.ini`'s `deps`:

```
[testenv]
deps = pytest
       pytest-cov
       pytest-cache
commands = {envpython} setup.py test`
```

after which things work:

```
$ tox -e py27
[...]
TOTAL                 169      7    96%   
[...]
103 passed, 15 skipped in 1.61 seconds 
```